pandas/core/frame.py

```diff
@@ -1918,6 +1919,8 @@ def get_value(self, index, col, takeable=False):
         """
         Quickly retrieve single value at passed column and index
 
+        .. deprecated:: 0.21.0
+
```

Comment:
Can you add here the alternative like in the deprecations message? (the "use '.at[]' or '.iat[]' instead")

(and the same for the other docstrings)